### DISCUSSION TITLE
Help creating fork

### DISCUSSION THREAD

**Discussion by n9jcv** [community/community#22269] [New to GitHub]
<p>I tried to create a fork of Marlin Marlin Firmware 1.1.8, when I click fork it does not ask me what version I want to fork.  It just assumes I want 1.1.9, which they label as 1.1.x.  I specifically want 1.1.8.</p>
<p>So I downloaded the zip of 1.1.8.  Then I dragged and dropped it into github desktop and created a repository.  So now it Github desktop, I have a 1.1.8 repository, but I do not believe it is a fork.  I made it a private repository.</p>
<p>My intention is to take 1.1.8 and repurpose this for a different project, my changes will not be wanted by the Marlin community and I have no intention of pushing them.  I also have no intention of pulling any further of their changes.  </p>
<p>Can you tell me easily how to create a 1.1.8 fork.  I would like it private until it works, and then I would make it public.  But I probably will not take any code in.  My intention is just to publish and make available to anyone who wants it, but I will not accept any changes.</p>
<p>Thank you</p>

**Comment by n9jcv** [comment, marked as answer]
<p>Maybe someone smarter than I with GitHub will correct me, but this is what I was able to find out.  </p>
<p>Github, does NOT require every release to be a branch.  It seems you can only fork branches and NOT all releases.  It is very wierd, as GitHub is the repository that holds the code even for each individual release.  Why not allow us to fork any release???</p>
<p>Anyway, I downloaded the zip of Marlin 1.1.8.  I did not use the GitHub desktop related to the 1.1.8 at all.  I then opened the project folder in Atom, and now I can build and download with 1.1.8.  So Techinically, I have my code to do what I want, like my fork, it just is not a register fork in GitHub.  When I am done coding, I plan to create a new repository calling this a fork and make it public for anyone who wants it, though I don’t think anyone will as these changes are just for me.</p>
<p>I hope this helps someone else</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/n9jcv">@n9jcv</a>,</p>
<p>Thanks for coming back to share what worked for you!</p>
